pkg/tfbridge/schema.go

```diff
@@ -1437,7 +1437,7 @@ func extractSchemaInputs(state resource.PropertyValue, tfs shim.Schema, ps *Sche
 	}
 }
 
-func extractInputsFromOutputs(oldInputs, outs resource.PropertyMap,
+func ExtractInputsFromOutputs(oldInputs, outs resource.PropertyMap,
```

Comment:
The implementation here dates back to #330 which appears incomplete with respect to end-user test cases and nailing down why the input extraction algorithm has the quirks it has. Because of this reimplementing for Plugin Framework seems like a substantial lift. So here I'm opting to re-export and reuse the existing implementation, and continue with it until we accrete more test cases or bug reports to refine.

It unfortunately creates some other inconsistencies, for example in how PF vs pre-PF bridge handle property naming, but so far appears OK in practice.